pkg/controller/ipam/ipam_test.go

```diff
@@ -168,6 +168,25 @@ var _ = ginkgo.Describe("Pod CIDRs", ginkgo.Ordered, func() {
 		}, gomega.Equal(expectedPodCIDRs3)))
 	})
 
+	ginkgo.It("should fail to update immutable fields", func() {
```

Comment:
where are these tests running?

should they be running here https://github.com/kubernetes-sigs/node-ipam-controller/actions/runs/10150448361/job/28067570270?pr=24?

it says

> ?   	sigs.k8s.io/node-ipam-controller/pkg/controller/ipam/test	[no test files]

Reply:
Tests output should be improved, agree. Ipam package run [output](https://github.com/kubernetes-sigs/node-ipam-controller/actions/runs/10150448361/job/28067570270?pr=24#step:4:145). [Output from](https://github.com/kubernetes-sigs/node-ipam-controller/actions/runs/10150448361/job/28067570270?pr=24#step:4:237) the controller for ClusterCIDR with the key `validate-immutable`

Reply:
not familiar with this test suite, thanks for the clarification